src/modules/50-assessments/components/AssessmentResults/AssessmentResults.tsx

```diff
@@ -0,0 +1,391 @@
+                                      highcharts={Highcharts}
+                                      options={getScoreComparisonChartOptions({
+                                        userScore,
+                                        questionOrgScore,
+                                        questionBenchMarkScore,
+                                        questionMaxScore
+                                      })}
+                                    />
+                                  </Layout.Horizontal>
+                                </Card>
+                              </>
+                            )
+                          }
+                        )
+                      : null}
+                  </Container>
+                </Card>
+              </Layout.Vertical>
+            </Layout.Vertical>
+          </Container>
+          <Dialog
+            {...DialogProps}
+            isOpen={isShareResultsModalOpen}
+            onClose={handleShareResultsModalClose}
+            title="Share Results"
+          >
+            <Layout.Horizontal flex={{ justifyContent: 'space-between' }}>
+              <TextInput value={window.location.href} className={css.fieldInput} />
+              <Button
+                variation={ButtonVariation.SECONDARY}
+                text={'Copy Link'}
+                onClick={() => copy2Clipboard(window.location.href)}
+                margin={{ right: 'small' }}
+              />
+            </Layout.Horizontal>
+            <Container padding={{ top: 'small' }}>
+              <HorizontalLineWithText text={'OR'} />
+            </Container>
+            <Layout.Horizontal flex={{ justifyContent: 'space-between' }}>
+              <Text padding={{ top: 'large', bottom: 'xxxlarge' }} margin={{ top: 'small' }}>
+                {'Download a PDF Report'}
+              </Text>
+              <Button
+                variation={ButtonVariation.SECONDARY}
+                text={'Download PDF'}
+                margin={{ right: 'small' }}
+                icon={'download'}
+                disabled
+              />
+            </Layout.Horizontal>
+          </Dialog>
+          <Dialog
+            {...InviteAssessmentModalDialogProps}
+            isOpen={isInviteAssessmentModalOpen}
+            onClose={handleInviteAssessmentModalClose}
+            title="Invite to take assessment"
+          >
+            <Layout.Horizontal flex={{ justifyContent: 'space-between' }} padding={{ bottom: 'xlarge' }}>
+              <MultiSelect
+                value={invitedUsers}
+                items={invitedUsers}
+                className={css.fieldInput}
+                placeholder={'Enter email addresses ...'}
+                onChange={(item: React.SetStateAction<MultiSelectOption[]>) => {
+                  setInvitedUsers(item)
+                }}
+              />
+              <Button
+                variation={ButtonVariation.SECONDARY}
+                text={'Send Invite'}
```

Comment:
are we not using strings here ?
```suggestion
                text={'Send Invite'}
```

Reply:
Moved to strings here and at other places also.